### PR TITLE
UsersGuide package

### PR DESCRIPTION
**Pull request by AtiyahElsheikh**
An initial package for UsersGuide following common guidelines. I have simply moved existing documentation to the UsersGuide package

### CODE REVIEW

**Comment by AtiyahElsheikh** [comment]
Sorry for the high number of commits. Only the last two commits can be considered for review. The other commits are due to merging from previous pull requests.

File: BioChem/UsersGuide/CopyRight.mo [hunk 1]
```diff
@@ -0,0 +1,7 @@
+within BioChem.UsersGuide;
+
+class CopyRight
+  extends Modelica.Icons.Information;
+  annotation(
+    Documentation(info = "<html><head></head><body>Copyright © 2005-2015 MathCore Engineering AB, Linköpings universitet and Modelica Association</body></html>"));
```
Comment (sjoelund) [inline BioChem/UsersGuide/CopyRight.mo:6]: You could probably bump that to 2021 and add your own name

File: BioChem/UsersGuide/References.mo [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
...215-220&nbsp;<br><br><b>A minimal cascade model for the mitotic oscillator involving cyclin and cdc2 kinase</b><br>Albert Goldbeter. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 9107-9111, October 1991 Cell Biology&nbsp;<br><br><b>Hierarchical modeling of diabetes</b><br>Elin Nyman. ISRN:LiU-IKE-EX-09/14. Linköping University 2009.&nbsp;<br><br><b>A Minimal Generic Model of Bacteria-Induced Intracellular Ca<sup>2+</sup>&nbsp;Oscillations in Epithelial Cells</b><br>Camilla Oxhamre, Agneta Richter-Dahlfors, Vladimir P. Zhdanov, and Bengt Kasemo. Biophysical Journal Volume 88 April 2005 2976-2981.&nbsp;<br><br><b>A mathematical model of metabolic insulin signaling.</b><br>A. Sedaghat, R, A. Sherman, and J. Quon, Michael. American Journal of Physiology - Endocrinology and Metabolism, 283:1048-1101, Jul 2002.&nbsp;<br><br><b>Modeling the cell division cycle: cdc2 and cyclin interactions</b><br>John J. Tyson. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 7328-7332, August 1991 Cell Biology&nbsp;<br><br><b>Modeling Feedback Loops of the Mammalian Circadian Oscillator</b><br>Becker-Weimann S, Wolf J, Herzel H, Kramer A. Biophysical Journal Volume 87 November 2004 3023-3034&nbsp;</p></body></html>"));
```
Comment (sjoelund) [inline BioChem/UsersGuide/References.mo:6]: Since it's HTML, perhaps add links to the papers while changing things?

Comment (AtiyahElsheikh) [reply BioChem/UsersGuide/References.mo]: Done

File: BioChem/UsersGuide/CopyRight.mo (hunk 1 above)
Comment (AtiyahElsheikh) [reply BioChem/UsersGuide/CopyRight.mo]: Done and have added your name as well under ReleaseNotes

File: BioChem/UsersGuide/References.mo [hunk 3]
```diff
@@ -3,5 +3,5 @@ within BioChem.UsersGuide;
...
...231708&amp;dswid=mainwindow\">link</a>)<br><br><b>A Minimal Generic Model of Bacteria-Induced Intracellular Ca<sup>2+</sup>&nbsp;Oscillations in Epithelial Cells</b><br>Camilla Oxhamre, Agneta Richter-Dahlfors, Vladimir P. Zhdanov, and Bengt Kasemo. Biophysical Journal Volume 88 April 2005 2976-2981 (<a href=\"https://www.sciencedirect.com/science/article/pii/S000634950573349X\">link</a>)<br><br><b>A mathematical model of metabolic insulin signaling.</b><br>A. Sedaghat, R, A. Sherman, and J. Quon, Michael. American Journal of Physiology - Endocrinology and Metabolism, 283:1048-1101, Jul 2002 (<a href=\"https://pubmed.ncbi.nlm.nih.gov/12376338/\">link</a>)<br><br><b>Modeling the cell division cycle: cdc2 and cyclin interactions&nbsp;</b><br>John J. Tyson. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 7328-7332, August 1991 Cell Biology&nbsp;<b>(<a href=\"https://pubmed.ncbi.nlm.nih.gov/1831270/\">link</a>)</b><br><br><b>Modeling Feedback Loops of the Mammalian Circadian Oscillator&nbsp;</b><br>Becker-Weimann S, Wolf J, Herzel H, Kramer A. Biophysical Journal Volume 87 November 2004 3023-3034 (<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1304775/\">link</a>)</p></body></html>"));
```
Comment (sjoelund) [inline BioChem/UsersGuide/References.mo:6]: researchgate? Why not the source? https://www.modelica.org/events/Conference2003/papers/h21_Larsdotter_Nilsson.pdf
And http://urn.kb.se/resolve?urn=urn:nbn:se:liu:diva-19076 is the "permanent link" according to the second URL

Comment (AtiyahElsheikh) [reply BioChem/UsersGuide/References.mo]: Done

**Review by sjoelund — APPROVED** [review]

**Comment by sjoelund** [comment]
Squash and merge will let you combine all commits into one

**Pull request merged**